cypress/specs/app.spec.ts

```diff
@@ -1,119 +1,103 @@
-const SNAPSHOT_DELAY = 500;
+// For ARIA-based checks
+const BE_SELECTED = ['have.attr', 'aria-selected', 'true'] as const;
+const BE_CHECKED = ['have.attr', 'aria-checked', 'true'] as const;
 
-describe('App', () => {
+afterEach(() => {
+  window.localStorage.clear();
+});
+
+describe('/mock', () => {
   beforeEach(() => {
     cy.visit('/mock');
-  });
-
-  afterEach(() => {
-    window.localStorage.clear();
+    cy.waitForStableDOM();
   });
 
   it('visualize 1D dataset as Line', () => {
-    cy.findByRole('treeitem', { name: 'nD_datasets' }).click();
-    cy.findByRole('treeitem', { name: 'oneD' }).click();
+    cy.selectExplorerNode('nD_datasets');
+    cy.selectExplorerNode('oneD');
```

Comment:
I've also added a couple of custom queries like in feature tests. Internally, they wait for the DOM to be stable.